src/font.h

```diff
@@ -0,0 +1,266 @@
+//  Copyright (C) 1999 by
+//  id Software, Chi Hoang, Lee Killough, Jim Flynn, Rand Phares, Ty Halderman
+//  Copyright (C) 2023 liPillON
```

Comment:
There were also some patches provided by @kraflab, right?

Reply:
I was not able to use those lumps (they weren't in a format recognized by SLADE3 out of the box) so I resorted to those available in the Eternity Engine repo.
https://github.com/team-eternity/eternity/tree/master/base/doom/res/graphics

Question: do you plan to include my tweaked STBR12* lumps, too?
If so, please note that the original files came from the EE repo as well.

Reply:
The tweaked progress bars are already included at the end of the `font.h` file.

Regarding the patches' origin, I am pretty sure that the ones from the Eternity repo are still the pristine ones as provided by Boom back then, but in a different format. We'd need a byte-by-byte comparison to be 100% sure, though.

Reply:
Alternate approach: Do you still know exactly which patches you modified? Then I could onyl replace these in `info.c`.

Reply:
new: DIG37, DIG43, DIG46
tweaked: STBR123, STBR124, STBR125, STBR126, STBR127
fixed: DIG1, DIG4, DIG5, DIG6, DIG7, DIGB, DIGI, DIGK, DIGQ, DIGR, DIGS, DIGV, DIGW, DIGX, DIGY

Reply:
> new: DIG37, DIG43, DIG46 

We already got these from DSDA some time ago.

> tweaked: STBR123, STBR124, STBR125, STBR126, STBR127 fixed: DIG1, DIG4, DIG5, DIG6, DIG7, DIGB, DIGI, DIGK, DIGQ, DIGR, DIGS, DIGV, DIGW, DIGX, DIGY

Alright, I will replace these with your variants.